driver-core/src/main/com/mongodb/internal/operation/ServerVersionHelper.java

```diff
@@ -25,22 +25,19 @@
 public final class ServerVersionHelper {
 
     public static final int MIN_WIRE_VERSION = 0;
-    public static final int FOUR_DOT_ZERO_WIRE_VERSION = 7;
     public static final int FOUR_DOT_TWO_WIRE_VERSION = 8;
     public static final int FOUR_DOT_FOUR_WIRE_VERSION = 9;
-    public static final int FIVE_DOT_ZERO_WIRE_VERSION = 12;
+    public static final int FIVE_DOT_ZERO_WIRE_VERSION = 13;
     public static final int SIX_DOT_ZERO_WIRE_VERSION = 17;
     public static final int SEVEN_DOT_ZERO_WIRE_VERSION = 21;
-    public static final int LATEST_WIRE_VERSION = SEVEN_DOT_ZERO_WIRE_VERSION;
+    public static final int EIGHT_DOT_ZERO_WIRE_VERSION = 25;
+    public static final int EARLIEST_WIRE_VERSION = FOUR_DOT_TWO_WIRE_VERSION;
```

Comment:
Not sure these are needed -`LATEST_WIRE_VERSION` is only used in tests and `EARLIEST_WIRE_VERSION` could be replaced by `MIN_WIRE_VERSION`.

Reply:
I removed EARLIEST_WIRE_VERSION and renamed MIN_WIRE_VERSION to UNKNOWN_WIRE_VERSION, which is more accurate.  I left LATEST_WIRE_VERSION since it's used quite a lot in tests and I wasn't sure what to replace it with.  Since this class is internal, this seems an ok spot to leave this constant.  What do you think?